logstash-core/lib/logstash/instrument/metric_store.rb

```diff
@@ -37,8 +39,8 @@ def get(*key_paths)
     # Take all the individuals `MetricType` and convert them to `Logstash::Event`
     #
     # @return [Array] An array of all metric transformed in `Logstash::Event`
-    def to_events
-      to_events_recursively(@store).flatten
+    def all_metrics
```

Comment:
not totally comfortable with a name like ```all_metrics``` for a class named ```MetricStore```, what do you think? might be named it simply ```all``` ?